doc/files/opensc.conf.5.xml.in

```diff
@@ -166,6 +166,16 @@ app <replaceable>application</replaceable> {
 						checked.
 				</para></listitem>
 			</varlistentry>
+			<varlistentry>
+				<term>
+					<option>disable_colors = <replaceable>bool</replaceable>;</option>
```

Comment:
I think having colors can also be helpful when the output is redirected to a file for later analysis.
Therefore: Is it possible to make the variable tri-state?
E.g. (following `ls`'s and `diff`'s `--colors` option)
* `never`: disabled
* `auto`: enabled on ttys only
* ` always`: enabled also on non-ttys / redirections / ...

Maybe for the tri-state variable a different name may be more appropriate, e.g. `log_colors`.